app/views/api/v1/twilio_numbers/index.json.jbuilder

```diff
@@ -0,0 +1,15 @@
+json.twilio_numbers  do
+  @locations.each do |location|
+    json.set!(location.twilio_number) do
+      json.uid location.uid
+
+      json.delivery_partner location.organisation
+
+      json.location location.title
+      json.location_postcode location.address.postcode
+
+      json.booking_location((location.booking_location || location).title)
```

Comment:
Should this `location.booking_location || location` logic sit in the location instead?